cirq-superstaq/cirq_superstaq/daily_integration_test.py

```diff
@@ -253,6 +253,22 @@ def test_dfe(service: css.Service) -> None:
     assert isinstance(result, float)
 
 
+def test_aces(service: css.Service) -> None:
```

Comment:
is this part of #708? /should that be merged first to simplify this pr?

Reply:
I accidentally merged that branch into this one :sob: . Will wait until that one is in before this